pkg/open-vm-tools/Dockerfile

```diff
@@ -13,4 +13,7 @@ ENTRYPOINT []
 CMD []
 WORKDIR /
 COPY --from=mirror /out/ /
+COPY scripts /etc/vmware-tools/scripts
 CMD ["/usr/bin/vmtoolsd"]
+
+LABEL org.mobyproject.config='{"pid": "host", "capabilities": ["CAP_SYS_BOOT"]}'
```

Comment:
pid=host is actually the default, although it probably still makes sense to have this explicitly.

Reply:
I thought that when I tested it was required, but I'll try without to verify.

Reply:
@justincormack yep, it looks like I don't need it. Should I leave `"pid": "host"` or leave it in?

Reply:
I prefer if it is explicit esp if it is really needed to work.

Reply:
@rn This requires access to the host pid space to kill `init`, so I'm quite certain this is the case. Thanks again for the merge.